rpcapd/rpcapd.manadmin.in

```diff
@@ -119,16 +175,21 @@ the service is started from the control panel
 .B \-i
 run in inetd mode (UNIX only).
 .TP
-.BI \-s " file"
+.BI \-s " config_file"
 save the current configuration to file
 .TP
-.BI \-f " file"
+.BI \-f " config_file"
 load the current configuration from file; all switches
 specified from the command line are ignored
 .TP
 .B \-h
 print this help screen
 .br
 .ad
+.SH "KNOWN BUGS"
```

Comment:
Umm, shouldn't that be filed as an issue?  That shouldn't be happening.

Reply:
I copied this part from the URL I mentioned. I have no idea whether this problem persists or not. I also don't have a FreeBSD system to test on.